src/components/views/rooms/EventTile.tsx

```diff
@@ -340,6 +340,9 @@ interface IProps {
     // displayed to the current user either because they're
     // the author or they are a moderator
     isSeeingThroughMessageHiddenForModeration?: boolean;
+
+    // Whether we should assume a smaller width and adjust layout to match
+    narrowMode?: boolean;
```

Comment:
If as per my other comment we raise it, we could insert it into the RoomContext and thus not have to weave it to every component